test/mocha/25-http-edv-client.js

```diff
@@ -393,20 +393,39 @@ describe('bedrock-edv-storage HTTP API - edv-client', () => {
       delta.content.should.eql(mockData.httpDocs.delta.content);
     });
     it('should get a document count when count is set to true', async () => {
-      let result;
+      let result1;
+      let result2;
+      let result3;
+
       let err;
       try {
-        result = await edvClient.find({
+        result1 = await edvClient.find({
           has: ['content.apples'],
           count: true,
           invocationSigner: capabilityAgent.getSigner(),
         });
+
+        result2 = await edvClient.find({
+          equals: [{'content.apples': mockData.httpDocs.beta.content.apples}],
+          count: true,
+          invocationSigner: capabilityAgent.getSigner(),
+        });
+
+        result3 = await edvClient.find({
+          equals: [{'content.foo': 'does-not-exist'}],
+          count: true,
+          invocationSigner: capabilityAgent.getSigner(),
+        });
       } catch(e) {
         err = e;
       }
       assertNoError(err);
-      result.should.be.an('object');
-      result.count.should.equal(4);
+      result1.should.be.an('object');
+      result1.count.should.equal(4);
```

Comment:
there's another useful assertion which is `result1.should.have.keys()` in the chai API if you want to check for exactly a certain set of object properties.  Please apply that to these results.